Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -154,6 +154,9 @@ define([
      * Part of the {@link Cesium3DTileContent} interface.
      */
     Batched3DModel3DTileContent.prototype.initialize = function(arrayBuffer, byteOffset) {
+        // Destroy expired resources if they exist.
+        destroyResources(this);
```

Comment:
It would be nice if the request included the current cesium time (timeline).  Might be out of the scope of this PR, but tiles should also 'expire' if you move backwards in time before the tile should be valid.  

Since streaming tiles looks to be far out the above changes allow us to create 3D tiles dynamically on the server side.